core/src/main/resources/lib/form/dropdownList.jelly

```diff
@@ -46,9 +46,9 @@ THE SOFTWARE.
         <j:if test="${attrs.help!=null}">
             <f:helpArea/>
         </j:if>
-        <div class="setting-main">
+        <div class="jenkins-select">
```

Comment:
credentials select has the text cut off:

![image](https://user-images.githubusercontent.com/21194782/142161943-a060bd20-4275-4eda-b151-e034753c0de3.png)

![image](https://user-images.githubusercontent.com/21194782/142162477-d422a548-2011-4856-93ce-073e45a2a11c.png)

Reply:
Not sure if this is the exact same page, but should be fixed

<img width="662" alt="image" src="https://user-images.githubusercontent.com/43062514/143933839-b7197eca-108f-43c2-8609-9b3485c19225.png">